agent/config/types.go

```diff
@@ -167,6 +168,18 @@ type Config struct {
 	// AWSVPCBlockInstanceMetdata specifies if InstanceMetadata endpoint should be blocked
 	// for tasks that are launched with network mode "awsvpc" when ECS_AWSVPC_BLOCK_IMDS=true
 	AWSVPCBlockInstanceMetdata bool
+
+	// OverrideAWSVPCLocalIPv4Address overrides the local IPv4 address chosen
+	// for a task using the `awsvpc` networking mode. Using this configuration
+	// will limit you to running one `awsvpc` task at a time. IPv4 addresses
```

Comment:
> Using this configuration will limit you to running one `awsvpc` task

Since we don't enforce this anywhere in the code (iiuc), may be we should add a `TODO` so that we don't lose sight of this? Even an issue would be fine.

Reply:
Do you want a TODO even if we don't have any short-term plans to change it?  If you statically-assign an IP address, you'll always be limited to one or else you'll have IP address conflicts and things won't work right.

Reply:
Hmm, may be the better thing to do here is to update the description for the `ECS_AWSVPC_ADDITIONAL_LOCAL_ROUTES ` field in the README file with some of what you have here as well?

Reply:
`ECS_AWSVPC_ADDITIONAL_LOCAL_ROUTES` won't limit you to running one `awsvpc` task; it just adds routes.  It's specifically the static IP assignment that is limiting here.

Reply:
So, `OverrideAWSVPCLocalIPv4Address` gets set just from the JSON config and nowhere else? If that's the case, it's probably fine to leave it be. I'm just looking for ways to highlight this behavior so that it doesn't lead to post `start-task` failures.

Reply:
Exactly.  The intent here is that this is meant to serve a very limited set of use-cases and is not really a generally useful feature, so it's excluded from the more-common way to configure the agent (environment variables).